python/ray/rllib/test/test_policy_evaluator.py

```diff
@@ -138,6 +139,7 @@ def testBasic(self):
                 "prev_rewards", "prev_actions"
         ]:
             self.assertIn(key, batch)
+            self.assertGreater(np.abs(np.mean(batch[key])), 0)
```

Comment:
do you only want to check `actions` here?

Reply:
I checked for everything since it's a good sanity check.